app/jobs/onceoff/recover_post_uploads.rb

```diff
@@ -0,0 +1,10 @@
+require_dependency "upload_recovery"
+
+module Jobs
+  class RecoverPostUploads < Jobs::Onceoff
+    def execute_onceoff(args)
+      grace_period = SiteSetting.purge_deleted_uploads_grace_period_days.days.ago
```

Comment:
give this a minimum and maximum I think ... minimum is say 30 days and maximum is 120. I do not wan this to do absolutely everything ...